packages/image/src/index.vue

```diff
@@ -29,13 +34,13 @@
 
 <script lang='ts'>
 
-import { defineComponent, computed, ref, onMounted, onBeforeUnmount, watch } from 'vue'
+import { t } from '@element-plus/locale'
+import { getScrollContainer, isInContainer, off, on } from '@element-plus/utils/dom'
+import isServer from '@element-plus/utils/isServer'
 import { isString } from '@vue/shared'
 import throttle from 'lodash/throttle'
-import isServer from '@element-plus/utils/isServer'
-import { on, off, getScrollContainer, isInContainer } from '@element-plus/utils/dom'
-import { t } from '@element-plus/locale'
-import ImageViewer from './image-viewer'
+import { computed, defineComponent, onBeforeUnmount, onMounted, ref, watch } from 'vue'
```

Comment:
Why not put modules of `vue` to the top

Reply:
Webstorm sort the import, I will revert it